test/trento_web/controllers/v1/suse_manager_controller_test.exs

```diff
@@ -257,4 +300,15 @@ defmodule TrentoWeb.V1.SUSEManagerControllerTest do
       |> assert_schema("UnprocessableEntity", api_spec)
     end
   end
+
+  defp generate_cves(n) do
+    Enum.map(
+      1..n,
+      fn _ ->
+        year = Enum.random(1_991..2_024)
+        id = Enum.random(0..9_999)
+        "CVE-#{year}-#{id}"
+      end
+    )
+  end
```

Comment:
Your call if this is fine like this or we want a factory for this data :+1:

Reply:
Honestly, I'd just keep it here until we need it somewhere else ^^

Reply:
No strong feeling :+1:

Reply:
Ah I wanted to and tried to do this but I couldn't get the factory to work because the `build` macro expects a `Map` to be returned, and this is just a `List` of `String`s (`[String.t()]`). Is there a way to do it?

Reply:
For future reference if anyone sees this - thanks for the link @dottorblaster!
https://github.com/beam-community/ex_machina?tab=readme-ov-file#non-map-factories